### ISSUE TITLE
The package is of bad quality warning on Ubuntu installation

### ISSUE THREAD

**Issue by alexxtasi** [FrancescoCeruti/linux-show-player#5]
hi there
I tried to install linux-show-player using the .deb file,
and this message came up
```The package is of bad quality```
following:
```
Lintian check results for /home/user/Downloads/linux-show-player_0.3.1_amd64.deb:
E: linux-show-player: arch-independent-package-contains-binary-or-object usr/lib/python3.4/dist-packages/_cffi_backend.cpython-34m.so
E: linux-show-player: arch-independent-package-contains-binary-or-object usr/lib/python3.4/dist-packages/rtmidi/_rtmidi.cpython-34m.so
E: linux-show-player: wrong-file-owner-uid-or-gid linux-show-player_0.3.1_amd64.deb 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/ 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/bin/ 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/bin/linux-show-player 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/ 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/ 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/ 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/JACK_Client-0.2.0.egg-info/ 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/JACK_Client-0.2.0.egg-info/PKG-INFO 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/JACK_Client-0.2.0.egg-info/SOURCES.txt 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/JACK_Client-0.2.0.egg-info/dependency_links.txt 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/JACK_Client-0.2.0.egg-info/installed-files.txt 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/JACK_Client-0.2.0.egg-info/requires.txt 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/JACK_Client-0.2.0.egg-info/top_level.txt 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/__pycache__/ 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/__pycache__/jack.cpython-34.pyc 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/_cffi_backend.cpython-34m.so 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/cffi-1.1.2.egg-info/ 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/cffi-1.1.2.egg-info/PKG-INFO 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/cffi-1.1.2.egg-info/SOURCES.txt 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/cffi-1.1.2.egg-info/dependency_links.txt 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/cffi-1.1.2.egg-info/entry_points.txt 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/cffi-1.1.2.egg-info/installed-files.txt 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/cffi-1.1.2.egg-info/not-zip-safe 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/cffi-1.1.2.egg-info/requires.txt 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/cffi-1.1.2.egg-info/top_level.txt 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/cffi/ 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/cffi/__init__.py 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/cffi/__pycache__/ 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/cffi/__pycache__/__init__.cpython-34.pyc 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/cffi/__pycache__/api.cpython-34.pyc 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/cffi/__pycache__/backend_ctypes.cpython-34.pyc 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/cffi/__pycache__/cffi_opcode.cpython-34.pyc 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/cffi/__pycache__/commontypes.cpython-34.pyc 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/cffi/__pycache__/cparser.cpython-34.pyc 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/cffi/__pycache__/ffiplatform.cpython-34.pyc 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/cffi/__pycache__/gc_weakref.cpython-34.pyc 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/cffi/__pycache__/lock.cpython-34.pyc 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/cffi/__pycache__/model.cpython-34.pyc 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/cffi/__pycache__/recompiler.cpython-34.pyc 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/cffi/__pycache__/setuptools_ext.cpython-34.pyc 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/cffi/__pycache__/vengine_cpy.cpython-34.pyc 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/cffi/__pycache__/vengine_gen.cpython-34.pyc 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/cffi/__pycache__/verifier.cpython-34.pyc 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/cffi/_cffi_include.h 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/cffi/api.py 1000/1000
E: linux-show-player: wrong-file-owner-uid-or-gid usr/lib/python3.4/dist-packages/cffi/backend_ctypes.py 1000/1000
```
I am using Ubuntu Gnome 15.04 64-bit

I pressed ```Ignore and install``` and it installed with no other messages

thanks

**Comment by FrancescoCeruti** [comment]
Hi,
the debian/ubuntu packaging is a bit complex, and require a lot of stuff I've intentionally left, to avoid losing my mind (actually I'm too lazy to look the docs of a distro family I don't use).

The next time I'll try to produce a better package.

I'm going to leave this issue open, as an invitation to someone to take care of the (deb) packaging  (probably the best way is to create a ppa on launchpad).

**Comment by nodiscc** [comment]
I plan on working on the Debian packaging when the the next version of linux-show-player is released. First step is to submit an RFP at https://www.debian.org/devel/wnpp/. @FrancescoCeruti Any ETA for the next release?

**Comment by FrancescoCeruti** [comment]
@nodiscc thanks :)
Before the end of the year a new version should be available.

**Comment by FrancescoCeruti** [comment]
Now that the new version is available I've submitted an RFP to the wnnp.
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=825028

**Comment by FrancescoCeruti** [comment]
@nodiscc @alexxtasi
So, I've invested a few days to create decent packages, and they are available with a new (0.4fix) release.
Maybe in the future I will consider submitting them on debian.
